creator-node/src/utils/decisionTree.ts

```diff
@@ -63,29 +69,51 @@ module.exports = class DecisionTree {
     this.tree.push(stage)
 
     if (log) {
-      this.printLastStage()
+      this.printLastStage(logLevel)
     }
   }
 
-  printTree() {
-    this._logInfo(`DecisionTree Full - ${JSON.stringify(this.tree, null, 2)}`)
+  printTree({ logLevel = 'info' }) {
```

Comment:
seems like we're switching between object args and positional args in this file. we should standardize but doesn't need to block merging for now

Reply:
good catch, fixed